packages/protocol/contracts/common/test/FixidityTest.sol

```diff
@@ -6,47 +6,91 @@ import "../FixidityLib.sol";
 contract FixidityTest {
   using FixidityLib for FixidityLib.Fraction;
 
-  function newFixed(uint256 a) external view returns (uint256) {
+  function newFixed(uint256 a)
```

Comment:
Most, if not all, of these functions should fit on one line each.

Reply:
+1, our line limit is 100 chars